parachain/runtime/tests/src/test_cases.rs

```diff
@@ -279,46 +287,6 @@ pub fn send_transfer_token_message_fee_not_enough<Runtime, XcmConfig>(
 				fee_amount,
 			);
 			// check err is NotHoldingFees
-			assert_err!(outcome.ensure_complete(), NotHoldingFees);
-		});
-}
-
-pub fn send_transfer_token_message_insufficient_fund<Runtime, XcmConfig>(
```

Comment:
Why do we remove this test?

Reply:
This is a test utility that's called from multiple tests, not a test itself. This was duplicated for each test case and I refactored it into a single [test utility that](https://github.com/Snowfork/snowbridge/pull/1067/files#diff-6b7b2955e408c054f0caa88f257fec221c05670262882c78a2794f54b3705ee8R247-L250).